mkdocs.yml

```diff
@@ -2,7 +2,11 @@ site_name: Zanata Server
 docs_dir: docs
 repo_name: GitHub
 repo_url: http://www.github.com/zanata/zanata-server
+site_favicon: images/favicon.ico
 theme: readthedocs
+extra_css:
+  - css/extra.css
+  - 'http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic'
```

Comment:
Have you try to bring in http://assets-zanata.rhcloud.com/master/assets/css/style.css and see what the page looks like? trying not to introduce another css.